.github/.env

```diff
@@ -1,5 +1,8 @@
 CARGO_TERM_COLOR=always
 NODE_VERSION=16.x
-PROGRAMS=["bubblegum/program"]
+PROGRAMS=["bubblegum"]
```

Comment:
@samuelvanderwaal @febo This change is breaking building programs. Sam, the tests were passing on your PR because the file-filters do not include the `.env` file so they just skipped all tests. I will create a new PR reverting that line and updating the file-filters (as I've done on other repos to avoid that change detection issue again).